refactor: replace aiohttp with httpx for future telegram-bot compatibility

NTGNguyen opened the pull request.

NTGNguyen committed:
refactor(deps): replace aiohttp with httpx for future telegram-bot compatibility(for get_data)

The telegram-bot library depends on httpx, so replacing aiohttp with
httpx reduces dependencies. This change ensures consistency and avoids
unnecessary packages.

NTGNguyen committed:
refactor(deps): replace aiohttp with httpx for future telegram-bot compatibility(telegram notification)

The telegram-bot library depends on httpx, so replacing aiohttp with
httpx reduces dependencies. This change ensures consistency and avoids
unnecessary packages.

KevinNitroG commented:
Edit the pr, just change from httpaio to httpx. Then in the desc of pr, write whys that @NTGNguyen

KevinNitroG commented:
Remove httpaio deps also bruh @NTGNguyen

KevinNitroG commented on src/cpn_core/get_data/check_phat_nguoi.py:
```diff
@@ -145,20 +145,23 @@ def __init__(self, *, timeout: float = 20) -> None:
         self,
         plate_info: PlateInfo,
     ) -> tuple[ViolationDetail, ...] | None:
+        print("nkdladwldakjdawndwadlljwadlwdhajdwadlwahdalwdhawd")
```
Remove odd test log

NTGNguyen replied:
Wait wat!!!! hêhe

KevinNitroG commented on src/cpn_core/get_data/csgt.py:
```diff
@@ -63,13 +62,15 @@ async def _get_phpsessid_and_captcha(self) -> tuple[str, bytes]:
-            captcha_img: bytes = await response.read()
+            phpsessid: str | None = response.cookies.get("PHPSESSID")
+            if not phpsessid:
+                raise ValueError("PHPSESSID cookie not found")
```
Have you handle this exception? If not necessary, just return or retry, but catch in retry!!

NTGNguyen committed:
chore: do as KevinNitroG's instructions

KevinNitroG committed:
chore(get-data): type annotate bytes

NTGNguyen committed:
chore: handle exception and add ssl context

- Handle exception for get data when using httpx
- Add ssl context to avoid error when access to csgt.vn

NTGNguyen committed:
chore: remove unused ssl context
